### PR TITLE
Refactor share for SSR support

### PR DESCRIPTION
**Pull request by lbiedinger**

### CODE REVIEW

File: pages/gallery/_.vue [hunk 1]
```diff
@@ -93,8 +89,7 @@
           { hid: 'description', name: 'description', content: this.description },
           { hid: 'og:title', property: 'og:title', content: this.title },
           { hid: 'og:image', property: 'og:image', content: this.shareMediaUrl },
-          { hid: 'og:type', property: 'og:type', content: 'article' },
-          { hid: 'og:url', property: 'og:url', content: this.canonicalURL }
```
Comment (mirjamverloop) [inline pages/gallery/_.vue]: This page now does not have a og:url. Is that correct?
There should be one, at least, according to https://ogp.me/#metadata .

File: pages/record/_.vue [hunk 2]
```diff
@@ -80,7 +80,7 @@
         <div class="card p-3 mb-3 bg-grey">
           <MediaActionBar
             :url="selectedMedia.about"
-            :share-url="canonicalURL"
+            :share-url="canonicalUrl"
```
Comment (mirjamverloop) [inline pages/record/_.vue]: Is this - and the other references in this file - still valid ? The SocialShare component no longer accepts shareUrl as a prop.

File: pages/gallery/_.vue (hunk 1 above)
Comment (lbiedinger) [reply pages/gallery/_.vue]: it does, but it doesn't update. This is still in progress

**Comment by lbiedinger** [comment]
Prefer  #503

**Pull request closed without merging**